src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.